### PR TITLE
feat: add alisls writer

### PR DESCRIPTION
**Pull request by sevennt**

### CODE REVIEW

**Comment by flycash** [comment]
Test failed

**Comment by sevennt** [comment]
> Test failed

Without Ali cloud sls AK configured, the test will be executed fail, I remove this test case later.

File: core/elog/ali/writer.go [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+	w.Sync()
+	return w, nil
+}
+
+func genLog(fields map[string]interface{}) *pb.Log {
+	l := &pb.Log{
+		Time:     proto.Uint32(uint32(time.Now().Unix())),
+		Contents: []*LogContent{},
+	}
```
Comment (askuy) [inline core/elog/ali/writer.go:115]: 这个slice的append是高频操作，可以提前分配slice长度

**Review by askuy — COMMENTED** [review]
slice需要提前分配length

**Pull request merged**